hack/lib/init.sh

```diff
@@ -23,6 +23,7 @@ network/v1 \
 oauth/v1 \
 openshiftcontrolplane/v1 \
 operator/v1 \
+operator/ingress/v1 \
```

Comment:
Is there any particular reason you're introducing a new package here, instead of just using `operator/v1` like all the others?

Reply:
The new type is in a different group, and mixing the group within the same package was causing me trouble. Maybe it's possible and I just didn't know the right incantations. If we _can_ mix groups within a single package, I can go back and try to figure it out. WDYT? cc @sttts

Reply:
@ironcladlou it was more of a question why you need that new group, mixing groups is definitely not recommended approach.

Reply:
Because we're moving an existing API group/type from one repo (cluster-ingress-operator) to this one. Using the sub-group was in hindsight probably a mistake, but now it's out there. The goal is to get the type into the API repo and out of the operator repo, not to change the API itself. Make sense?

Reply:
@soltysh @sttts @damemi Let me know how I should proceed here. Either this is ready to merge or I need to restructure in a way that satisfies everyone but preserves our `ingress.operator.openshift.io` group.

Reply:
Don't use operator/ingress, but operatoringress. This flat directory structure is a must.

Reply:
Fixed, thanks